redhat-ipaas-dev-single-tenant.yml

```diff
@@ -365,6 +370,8 @@ objects:
             value: "true"
           - name: JAVA_OPTIONS
             value: "-Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStore=/tls-keystore/openshift-truststore.jks -Duser.home=/tmp"
+            - name: ENDPOINTS_TEST_SUPPORT_ENABLED
```

Comment:
What happens in the app when this environment variable is set ? Is this some infrastructure thing (like the other envs) or more an app thing ? If the latter, I think its better to put it into the application.yaml which is mounted as configmap.